TFNetworkLayer.py

```diff
@@ -3403,9 +3403,13 @@ def get_out_data_from_opts(cls, name, value=0., dtype=None, with_batch_dim=False
         dtype = "bool"
       else:
         raise TypeError("cannot handle value %r of type %r" % (value, type(value)))
+
+    import numpy
```

Comment:
Make that import at the beginning of the function, and avoid the extra empty lines around it.